src/parser/paladin/holy/CombatLogParser.test.js

```diff
@@ -4,4 +4,5 @@ import CombatLogParser from './CombatLogParser';
 describe('Holy Paladin integration test', integrationTest(
   CombatLogParser,
   'holy-paladin-example',
+  'B11_test',
```

Comment:
Looking at the paramters of the `integrationTest` function I don't think this is how you add another integration test. I think you have to copy-paste the entire describe clause. I think this is the `suppressLog` parameter.

Do note that you're not required to add a new integration test for most features. Using the existing test often gives us sufficient confidence that things don't break on changes. Having said that, adding a test that covers an entirely different situation might still be valuable. You will have to update the snapshot however. This can be achieved by running `yarn test` and pressing `u` to update them. This will write new `.snap` files, which you can add to your branch. Try to review if the diff of those files match what you would expect. 

What is B11?

Reply:
Ok, I am gonna commit without the B11 test in it.  I am not entirely sure what I should expect the results to be as part of the integration testing.  

In my BeaconUptime.js I have a large block of comments indicating all the tests I ran to verify that the new feature is working.  B11 refers to a specific set of logs to test a specific scenario:

![image](https://user-images.githubusercontent.com/23178481/71047652-5300f400-20f1-11ea-9938-2ff9847291d4.png)